scalalib/src/ZincWorkerModule.scala

```diff
@@ -69,8 +69,9 @@ trait ZincWorkerModule extends mill.Module{
     instance.asInstanceOf[mill.scalalib.api.ZincWorkerApi]
   }
 
-  def scalaCompilerBridgeSourceJar(scalaVersion: String,
-                                   scalaOrganization: String) = {
+  def scalaCompilerBridgeJar(scalaVersion: String,
```

Comment:
Please add a documentation comment to this method which in particular should describe what it returns.